TAScheduler/models.py

```diff
@@ -23,16 +23,29 @@ class Section(models.Model):
     credits = models.IntegerField()
 
 
+# noinspection DuplicatedCode
 class TA(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
     sections = models.ManyToManyField(Section)
 
 
+# noinspection DuplicatedCode
+class TAToSection(models.Model):
+    ta = models.ForeignKey(TA, on_delete=models.CASCADE, null=True)
+    section = models.ForeignKey(Section, on_delete=models.CASCADE, null=True)
+
+
+# noinspection DuplicatedCode
 class Instructor(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
     sections = models.ManyToManyField(Section)
 
 
+# noinspection DuplicatedCode
```

Comment:
Same as my previous comment ^ instructor -> section is not a junction table (each section, a lecture in this case, can only hold 1 instructor). instructor -> course needs a junction table